src/components/vote/proposals/LayoutProposals.vue

```diff
@@ -59,8 +59,10 @@
             :value="item[column.key]"
           >
             {{ item[column.key] }}
-          </slot>
+          </slot>          
```

Comment:
you can avoid committing this when you run `yarn lint` it will also auto format

Reply:
yeah I think this linting task can be executed automatically when pushing.

Reply:
yes, we can do that but linting takes quite long

Reply:
I recommend setting up your IDE to lint/prettier on save.

Reply:
as @H34D said, linting task took heavy long. I handled it manually.